presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1169,4 +1178,152 @@ public void testInsertForDefaultColumn()
     }
 
     protected abstract TestTable createTableWithDefaultColumns();
+
+    @Test(dataProvider = "testDataMappingSmokeTestDataProvider")
+    public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
+    {
+        String prestoTypeName = dataMappingTestSetup.getPrestoTypeName();
+        String sampleValueLiteral = dataMappingTestSetup.getSampleValueLiteral();
+        String highValueLiteral = dataMappingTestSetup.getHighValueLiteral();
+
+        String tableName = "test_data_mapping_smoke_test_" + prestoTypeName.replaceAll("[^a-zA-Z0-9]", "_");
+
+        Runnable setup = () -> {
+            String createTable = format("CREATE TABLE %s(id bigint, value %s)", tableName, prestoTypeName);
+            assertUpdate(createTable);
+            assertUpdate(
+                    format("INSERT INTO %s VALUES (10000, NULL), (10001, %s), (99999, %s)", tableName, sampleValueLiteral, highValueLiteral),
```

Comment:
10000 -> 1, 10001 -> 2, 99999 -> 3

Reply:
Why?

Reply:
That way it is a bit easier. It is just nit.

Reply:
IMO contrary -- 10001 or 99999 are easier to find than 1,2

Reply:
This depends on what you are optimizing for.